Validation: description-missing fixes. (#242)
MicrosoftDocs/bingmaps-docs: BingMaps/v8-web-control/map-control-concepts/search-module-examples/index.md

---
title: "Search Module Examples | Microsoft Docs"

> description: Describes the Search module and provides a list of code examples and related topic articles to the Search module. 

Reviewer: ```suggestion
description: Describes the Search module and provides links to code examples and related topic. 
```
